<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBuscarCliente</class>
 <widget class="QDialog" name="DialogBuscarCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>369</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Buscador de Clientes</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1">
      <widget class="QDateEdit" name="dateEdit_FechaVencimiento">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="date">
        <date>
         <year>2017</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_FechaPago">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Fecha de Pago</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_Apellido">
       <property name="text">
        <string>Apellido:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEdit_ID">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_FechaVencimiento">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Fecha de Vencimiento</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit_Dni">
       <property name="inputMethodHints">
        <set>Qt::ImhNone</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_Dni">
       <property name="text">
        <string>DNI:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Nombre">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEdit_Nombre">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEdit_Apellido">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDateEdit" name="dateEdit_FechaPago">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="date">
        <date>
         <year>2017</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_Aceptar">
       <property name="text">
        <string>Aceptar</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_Cancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_Buscar">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_ModificarDatos">
       <property name="text">
        <string>Modificar Datos</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButton_VerFicha">
       <property name="text">
        <string>Ver Ficha</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QGraphicsView" name="graphicsView_foto">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_Sexo">
       <property name="text">
        <string>Sexo</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="radioButton_SexoF">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>F</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="radioButton_SexoM">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>M</string>
       </property>
       <property name="autoExclusive">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
